x-pack/plugins/cases/public/components/user_actions/severity.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiHealth } from '@elastic/eui';
+import React from 'react';
+import { SeverityUserAction } from '../../../common/api/cases/user_actions/severity';
+import { SET_SEVERITY_TO } from '../create/translations';
+import { createCommonUpdateUserActionBuilder } from './common';
+import { UserActionBuilder, UserActionResponse } from './types';
+import { severities } from '../severity/config';
+
+const getLabelTitle = (userAction: UserActionResponse<SeverityUserAction>) => {
+  const severity = userAction.payload.severity;
+  const severityData = severities[severity];
```

Comment:
Can we do a sanity check here if `severityData` is `undefined` and return `null` if so?

Reply:
done!